env-templates/exawind_ascicgpu_tests.yaml

```
@@ -5,9 +5,9 @@ spack:
     unify: when_possible
```

```suggestion
    unify: false
```
This actually may be your issue.  `unify:when_possible` makes an effort to reduce the DAG as much as possible. It is a beta feature so I may have added it too soon.

---

working after changed to `unify: false`